arp_scripts/submit_smd.sh

```diff
@@ -183,7 +182,8 @@ fi
 # Source the right LCLS-I/LCLS-2 stuff based on the experiment name
 if echo $LCLS2_HUTCHES | grep -iw $HUTCH > /dev/null; then
     echo "This is a LCLS-II experiment"
-    source $SIT_ENV_DIR/sw/conda2/manage/bin/psconda.sh
+    ABS_PATH=`echo $MYDIR | sed  s/arp_scripts/lcls2_producers/g`
```

Comment:
Should the `sdf_setup.sh` script change the permissions on this directory as well (seems to still only modify `producers`)